native/screens/UserProfile.jsx

```diff
@@ -138,14 +152,14 @@ const OwnerOptions = ({ setReviewStatus }) => {
           handleTextChange(true, 'renderDecisionModal', setReviewStatus);
           handleTextChange(true, 'ownerDecision', setReviewStatus);
         }}
-      />
+        />
       <Button
         title="Reject Request"
         onPress={() => {
           handleTextChange(true, 'renderDecisionModal', setReviewStatus);
           handleTextChange(false, 'ownerDecision', setReviewStatus);
         }}
-      />
+        />
```

Comment:
these look 1 tab indentation too deep, open and close brackets should be parellel
<Open bracket
   -thing 1
   -thing 2
   -thing 3
/> close bracket